components/SearchBox/components/NoResults.tsx

```diff
@@ -0,0 +1,16 @@
+import type { FC } from 'react';
+
+import styles from './index.module.css';
+
+type NoResultsProps = {
```

Comment:
```suggestion
type NoResultsProps = { searchTerm: string; };
```

Inline types whenever possible.

Reply:
Do you mean moving from this:

```ts
type NoResultsProps = {
  searchTerm: string;
};

export const NoResults: FC<NoResultsProps> = props => (
  <div className={styles.noResultsContainer}>
    No results for&nbsp;
    <span className={styles.noResultsTerm}>{`"${props.searchTerm}"`}</span>.
  </div>
);
```

to this?

```ts
export const NoResults: FC<{  searchTerm: string }> = props => (
  <div className={styles.noResultsContainer}>
    No results for&nbsp;
    <span className={styles.noResultsTerm}>{`"${props.searchTerm}"`}</span>.
  </div>
);

```

Reply:
Nope. Just avoid using multi-line constants when they can be one line. i.e., only have line breaks if prettier demands them.

Reply:
ooh got it. alright then